src/components/map/layers/NetworkLayers.tsx

```diff
@@ -125,23 +119,28 @@ export default class NetworkLayers extends Component<INetworkLayersProps> {
         };
     }
 
-    private addNodeFromClickEvent = (clickEvent: any) => {
+    private addNodeFromInitialClickEvent = (clickEvent: any) => {
         const properties =  clickEvent.sourceTarget.properties;
+        if (properties.soltyyppi !== NodeType.STOP) return;
+
         // TODO: Use factory call instead of service call because
         // geojson / geojsonManual are not found from properties
         NodeService.fetchNode(properties.soltunnus).then((node) => {
             if (node) {
                 this.props.newRoutePathStore!.addNode({
-                    id: node.id,
+                    nodeId: node.id,
```

Comment:
It seems we have a bit different coding styles for different objects. My preference is to always set the primary key (id) property called 'id'. So, Node.id, Stop.id, Line.id, Route.id. And it seems this is the way that it is currently done for 4 of our models. 2 of our models use the style that you now use here, for example: (route.routeId, line.lineId). We should use the same style for all models, so, I don't see the reason for changing this to 'nodeId'

Reply:
Yes we should use either id or nodeId